README.md

```diff
@@ -14,7 +14,9 @@
 
 # Tegel Design System
 
-https://tegel.scania.com/
+Official website: https://tegel.scania.com/
+
+Prod Storybook: https://tegel-storybook.netlify.app/
```

Comment:
Consider changing from "Prod" to "Production" to stay away from abbreviations and to be more inclusive.

Reply:
Or just "Storybook"?

Reply:
Yep, I would also go with just "Storybook", shorter and simpler.